test/e2e/constants.go

```diff
@@ -0,0 +1,46 @@
+package e2e
+
+const (
+	// OLSNameSpace is the namespace where the operator is deployed
+	OLSNameSpace = "openshift-lightspeed"
+	// OperatorDeploymentName is the name of the operator deployment
+	OperatorDeploymentName = "lightspeed-operator-controller-manager"
+	// LLMTokenEnvVar is the environment variable containing the LLM API token
+	LLMTokenEnvVar = "LLM_TOKEN"
+	// LLMTokenFirstSecretName is the name of the first secret containing the LLM API token
+	LLMTokenFirstSecretName = "llm-token-first" // #nosec G101
+	// LLMTokenSecondSecretName is the name of the second secret containing the LLM API token
+	LLMTokenSecondSecretName = "llm-token-second" // #nosec G101
+	// LLMApiTokenFileName
+	LLMApiTokenFileName = "apitoken"
+	// LLMDefaultProvider
+	LLMDefaultProvider = "openai"
+	// LLMProviderEnvVar is the environment variable containing the LLM provider
+	LLMProviderEnvVar = "LLM_PROVIDER"
+	// LLMDefaultModel is the default model to use
+	LLMDefaultModel = "gpt-3.5-turbo"
+	// LLMAlternativeModel is the alternative model to test model change
+	LLMAlternativeModel = "gpt-4-1106-preview"
```

Comment:
Let us name this as OPENAI LLM providers + Models . we will have multiple providers  in actual e2e config 
 in CI pipeline .@raptorsun